src/App.css

```diff
@@ -1,38 +1,50 @@
-.App {
+.app {
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+}
+
+h1 {
   text-align: center;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.nav-container {
+  margin-top: auto;
+  padding-top: 3rem;
+  padding-bottom: 3rem;
+  text-align: center;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.nav-container nav {
+  max-width: 50rem;
+  margin-left: auto;
+  margin-right: auto;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.nav {
   display: flex;
-  flex-direction: column;
-  align-items: center;
+  flex-wrap: wrap;
   justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  align-items: center;
+  gap: 0 0.75rem;
+  margin: 0;
+  padding: 0;
+  font-size: 1rem;
+  font-weight: 700;
+  list-style: none;
 }
 
-.App-link {
-  color: #61dafb;
+.nav__link {
+  margin: 0.15rem 0.5rem;
+  padding: 0.3rem 0.4rem 0.2rem;
+  color: black;
+  font-size: 1rem;
+  line-height: 1.5;
+  cursor: pointer;
+  text-decoration: none;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+.active {
```

Comment:
Today I learned: I didn't know `NavLink` automatically adds the `active` class. Very handy.

Reply:
And today I learned about the `gap` property!